docs/ChangeLog.md

```diff
@@ -14,7 +28,7 @@
 
 * This is the 2.1.0 release. The Splunk Operator for Kubernetes is a supported platform for deploying Splunk Enterprise with the prerequisites and constraints laid out [here](https://github.com/splunk/splunk-operator/blob/master/docs/README.md#prerequisites-for-the-splunk-operator)
 
-* This release depends upon changes made concurrently in the Splunk Enterprise container images. You should use the splunk/splunk:9.0.2 image with it
+* This release depends upon changes made concurrently in the Splunk Enterprise container images. You should use the splunk/splunk:9.0.3-a1 image with it
```

Comment:
This is not correct. 2.1.0 uses 9.0.2 and not 9.0.3-a1. Please revert it.

Reply:
Sorry my bad on this as well.